gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -427,25 +428,20 @@ private void cancelDag(String dagToCancel) throws ExecutionException, Interrupte
         List<DagNode<JobExecutionPlan>> dagNodesToCancel = this.dagToJobs.get(dagToCancel);
         log.info("Found {} DagNodes to cancel.", dagNodesToCancel.size());
         for (DagNode<JobExecutionPlan> dagNodeToCancel : dagNodesToCancel) {
-          cancelDag(dagNodeToCancel);
+          cancelDagNode(dagNodeToCancel);
```

Comment:
Will dagNodesToCancel include jobs that finished successfully? Or only jobs currently running?

Reply:
dagToJobs contains only running jobs, so it cancels only running jobs

Reply:
Ack.